packages/legacy-scripting-runner/index.js

```diff
@@ -530,6 +530,7 @@ const legacyScriptingRunner = (Zap, zcli, input) => {
             result = method(convertedBundle, optionalCallback);
           } catch (err) {
             reject(err);
+            return;
```

Comment:
A tricky one-liner fix. We should stop immediately if the scripting method throws an error. Without the `return` after the `reject`, the program would keep running using the undefined `result`, which leads to an error like `TypeError: Cannot read property 'files' of undefined` happened at this [line](https://github.com/zapier/zapier-platform/blob/9e2a6945b3a8edf95b030e599e5afb016431df8c/packages/legacy-scripting-runner/index.js#L166).